pkg/sdk/alert.go

```diff
@@ -0,0 +1,421 @@
+package sdk
```

Comment:
this file should be names "alerts" for consistency

Reply:
Ok, I was lookin

Reply:
Ok, I was lookin

Reply:
Ok, I was looking at masking_policy

Reply:
Ok, I was looking at masking_policy

Reply:
Ok

Reply:
Ok

Reply:
Ok